docs/federated_a_crd.md

```diff
@@ -0,0 +1,275 @@
+# Federate a CRD in the target cluster
+
+If you want to federate a CRD into your target cluster then the following example will help you achieve the same.
+
+## Getting Started
+
+These instructions will provide the process of federating a CRD across the target clusters.
+
+### Prerequisites
+
+The federation v2 suppports installation for both cluster-scoped and namespace-scoped. In user guide, we are following `cluster-scoped` federation and deploying federation controller in the namespace `federation-system`. Also, you will be creating the two clusters i.e. `cluster1` and `cluster2` and federation v2 will be installed on `cluster1`.
+
+### Example CRD to federate
+
+Let's say you want to federate a CRD of the type `Bar` then use the following `bar_crd.yaml`.
+
+```
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: bars.example.io
+spec:
+  group: example.io
+  versions:
+    - name: v1
+      served: true
+      storage: true
+  scope: Namespaced
+  validation:
+    openAPIV3Schema:
+      properties:
+        apiVersion:
+          type: string
+        kind:
+          type: string
+        metadata:
+          type: object
+        spec:
+          properties:
+            template:
+              type: object
+          type: object
+        status:
+          type: object
+  names:
+    plural: bars
+    singular: bar
+    kind: Bar
+```
+
+### Install the target CRD in all clusters
+
+Make sure to install target CRDs on all member clusters otherwise this example will not work as expected.
+
+```
+$ kubectl --validate=false apply -f bar_crd.yaml --context=cluster1
+customresourcedefinition.apiextensions.k8s.io/bars.example.io created
+
+$ kubectl --validate=false apply -f bar_crd.yaml --context=cluster2
+customresourcedefinition.apiextensions.k8s.io/bars.example.io created
+```
+
+### Create the Federation API for your CRD
+
+Now that we've created the CRD in all the clusters we want to federate it to, let's create the federation API for that CRD. The federation API for your CRD is distinct from the CRD itself and is the API surface that declares what the state that should be spread to different clusters is.
+
+There are three pieces of the federation API for a type. <br>
```

Comment:
I would rename the html breaks and just use a bulleted list in markdown instead.